annotated_test.go

```diff
@@ -444,6 +444,22 @@ func TestAnnotatedWrongUsage(t *testing.T) {
 		)
 		assert.Contains(t, app.Err().Error(), "embeds a dig.In", "expected error when result types were annotated")
 	})
+
+	t.Run("invalid group option", func(t *testing.T) {
+		t.Parallel()
+
+		app := NewForTest(t,
+			fx.Provide(
+				fx.Annotated{
+					Group: "foo,soft",
+					Target: func() string {
+						return "sad times"
+					},
+				},
```

Comment:
Can you use `fx.Annotate` instead? We're trying to deprecate `Annotated`

```suggestion
				fx.Annotate(func() string {
						return "sad times"
				}, fx.ResultTags(`group:"foo,soft"`))
```